publication-commons/src/main/java/no/unit/nva/publication/doi/dto/PublicationDate.java

```diff
@@ -0,0 +1,112 @@
+package no.unit.nva.publication.doi.dto;
+
+import static java.util.Objects.nonNull;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.Objects;
+import nva.commons.utils.JacocoGenerated;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class PublicationDate {
+
+    public static final String YEAR_JSON_POINTER = "/publication_date/year";
```

Comment:
This is a runtime dependency to the model. Maybe better to have a constructor from DAO and then have compile-itme dependency to the model. This way it will be easy to spot when the pattern has changed. Now we will only find out from the DLQ or from some kind of bug

Reply:
So this is not to be confused by the nva-datamodel-java, this is not in use here.

Reply:
Hence package name `publication.doi.dto`